src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -1,373 +1,58 @@
 <template>
   <div id="filing-history-list">
     <AddCommentDialog
-      :dialog="addCommentDialog"
-      :filing="currentFiling"
+      :dialog="isAddCommentDialog"
+      :filing="getCurrentFiling"
       @close="hideCommentDialog($event)"
       attach="#filing-history-list"
     />
 
     <DownloadErrorDialog
-      :dialog="downloadErrorDialog"
-      @close="downloadErrorDialog=false"
+      :dialog="isDownloadErrorDialog"
+      @close="mutateDownloadErrorDialog(false)"
       attach="#filing-history-list"
     />
 
     <LoadCorrectionDialog
-      :dialog="loadCorrectionDialog"
-      @exit="loadCorrectionDialog=false"
+      :dialog="isLoadCorrectionDialog"
+      @exit="mutateLoadCorrectionDialog(false)"
       attach="#filing-history-list"
     />
 
     <FileCorrectionDialog
-      :dialog="fileCorrectionDialog"
-      @exit="fileCorrectionDialog=false"
-      @redirect="redirectThisFiling($event)"
+      :dialog="isFileCorrectionDialog"
+      @exit="mutateFileCorrectionDialog(false)"
+      @redirect="redirectFiling($event)"
       attach="#filing-history-list"
     />
 
-    <!-- Alternate Loading Spinner -->
-    <v-fade-transition>
-      <div class="loading-container grayed-out" v-show="isBusy">
-        <div class="loading__content">
-          <v-progress-circular color="primary" size="50" indeterminate />
-          <div class="loading-msg white--text">Fetching data</div>
-        </div>
-      </div>
-    </v-fade-transition>
-
     <!-- Court order notification -->
     <v-card v-if="hasCourtOrders" class="my-6 pa-6" elevation="0">
       <v-icon>mdi-gavel</v-icon>
-      <span class="ml-1">Court order(s) have been filed on this company. Review the filing history for impacts
-        to business information.</span>
+      <span class="ml-1">Court order(s) have been filed on this company. Review the filing history for
+        impacts to business information.</span>
     </v-card>
 
-    <div class="scrollable-container">
-      <v-expansion-panels v-if="showHistoryPanel" v-model="panel">
-        <v-expansion-panel
-          class="align-items-top filing-history-item px-6 py-5"
-          v-for="(filing, index) in historyItems"
+    <div class="scrollable-container" v-if="showHistoryPanel">
+      <v-expansion-panels :value="getPanel">
```

Comment:
What do you think about tucking these two lines into the `FilingTemplate`?  I think this makes it look cleaner and easier to read.  The `showHistoryPanel` then becomes redundant since if getFilings doesn't return any data, the `<component>` won't get rendered.

Reply:
I think it is the FilingHistoryList.vue's job to display the list vs the no-results message.

Also I don't see how I can move the expansion panels component inside the template, since the template is iterated upon (many copies of "panel" inside 1 "panels").

What am I not seeing?

Reply:
Whoops, that's my mistake, you're absolutely right.  That makes no sense!